pblast_project.py

```diff
@@ -6,24 +6,31 @@
 # get the first frame of the timeline
 start_time = cmds.playbackOptions(query=True, minTime=True)
 
-#name the file
-path = r"C:\Users\Natalia\Desktop\prueba"
-file_name = f"{path}\\prueba"
+def create_sequence_of_pngs(path, file_name, width1, height1):
+    """
+    Args:
+        path(str): stores the name of the folder you want to store your pngs in
+        file_name(str): stores the name of the file for each png
+        width(int): the horizontal size of the image
+        height(int): the vertical size of the image
+    """
```

Comment:
```suggestion
        path (str): The path to the folder you want to store your pngs in
        file_name (str): The name of the file for each png
        width (int): The horizontal size of the image
        height (int): The vertical size of the image
    """
```

Una sugerencia de formato y para simplificar la oraciones un poco.
(Puedes aplicar la sugerecia directamente con el boton y lo hara como un comit nuevo en tu branch)

Tambien agregaria una descipcion en el dosctring para que sea mas claro cual sera el resultado.

Reply:
no me deja con el boton, me dice: Branching suggestions must be done from the files tab

Reply:
ah que raro, lo intente por curiosidad y si me dejo hacerlo a mi jaja entonces ya lo hice por, disculpa